src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main
+      class="calendar calendar--start-day--mon calendar--month-length">
```

Comment:
The 'month-length' modifier is missing a value (e.g., '28', '29', '30', or '31'). Add the desired value for the 'month-length' modifier.